CedarJava/src/main/java/com/cedarpolicy/AuthorizationEngine.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2022-2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We can change all of these to just "Copyright Cedar Contributors"

Reply:
Yes, I saw that in release/4.0.0. I think it is better than keeping track of these dates. However, I believe the current branch uses that copyright format so I wanted to keep things consistant